soul-spring-boot-starter/soul-spring-boot-starter-sync-data-center/soul-spring-boot-starter-sync-data-nacos/pom.xml

```diff
@@ -42,5 +42,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba.nacos</groupId>
+            <artifactId>nacos-client</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Please check the dependencies in `pom.xml`. The `soul-spring-boot-starter-sync-data-nacos` depends `soul-sync-data-nacos` and the `soul-sync-data-nacos `depends `nacos-client`

Reply:
yeah,the `provided` just make build pass